pubspec.yaml

```diff
@@ -66,7 +66,12 @@ flutter:
 
   # To add assets to your application, add an assets section, like this:
   assets:
-    - assets/

```

Comment:
this is fine no need to put individual files names here, just putting up folder path will be enough